learn/sherif/main_test/src/main/java/frc/robot/Constants.java

```diff
@@ -16,8 +16,17 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-    public static final int FRONT_MOTOR = 0;
-    public static final int BACK_MOTOR = 1;
-    public static final int CONTROLLER = 10;
-    public static final int LEFT_STICK_BUTTON = 1;
+    // Drive Motors
+    public static final int LEFT_FRONT = 0;
+    public static final int LEFT_MID = 1;
+    public static final int LEFT_END = 2;
+    public static final int RIGHT_FRONT = 20;
+    public static final int RIGHT_MID = 21;
+    public static final int RIGHT_END = 22;
```

Comment:
Technically, the three motors are combined in a gearbox so it's not exactly front/mid/end (it's not a big deal but just something to keep in mind)

Reply:
Rename these constants with `DRIVE_` at the front to make it clear what part of the robot they are related to.